rpc_util.go

```diff
@@ -213,35 +249,92 @@ func Peer(p *peer.Peer) CallOption {
 //
 // By default, RPCs are "Fail Fast".
 func FailFast(failFast bool) CallOption {
-	return beforeCall(func(c *callInfo) error {
-		c.failFast = failFast
-		return nil
-	})
+	return FailFastCallOption(failFast)
+}
+
+// FailFastCallOption is a CallOption for indicating whether an RPC should fail
+// fast or not.
+// This is an EXPERIMENTAL API.
+type FailFastCallOption bool
```

Comment:
We have some CallOptions that are structs and some that are not.  We should unify these since they are now exported.  The struct gives us a little better extensibility, so unless there's a compelling reason to use non-structs, I would prefer structs.

Reply:
That totally makes sense. The methods only made sense when relying on caller to do an interface-type-assertion to call it. With exported types, this is much better.